pyOCD/transport/cmsis_dap.py

```diff
@@ -65,6 +65,12 @@
                  32: CSW_SIZE32
                  }
 
+DAP_MODE_SWD = 1
+DAP_MODE_JTAG = 2
+
+STICKORUN = 0x00000002
```

Comment:
I'm not sure I understand what you are recommending here @0xc0170. The first two values are response values to the DAP_Connect command specified in the DAP protocol. Can you make a small code snippet of what you think it should look like? Thanks!